src/Type/TypeUtils.php

```diff
@@ -182,15 +184,6 @@ public static function getEnumCaseObjects(Type $type): array
 		return self::map(EnumCaseObjectType::class, $type, false);
 	}
 
-	/**
-	 * @internal
-	 * @return ConstantArrayType[]
-	 */
-	public static function getOldConstantArrays(Type $type): array
-	{
-		return self::map(ConstantArrayType::class, $type, false);
-	}
-
```

Comment:
the BC checker does not like the removal, which makes sense I guess. should it stay then?

Reply:
The BC checker isn't aware of `@internal`. Yeah, probably better to leave the method in the class, I'll remove it in 2.0 :) Also you can add `@deprecated` to it too so I can find it. THanks :)